front/src/main/java/org/example/services/BankAccountService.java

```diff
@@ -2,41 +2,44 @@
 
 import com.banking.api.BankAccountOpenFeignClient;
 import com.banking.dto.BankAccountDto;
-import java.util.List;
 import java.util.UUID;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
 @RestController
 @RequestMapping("/front/accounts")
 @RequiredArgsConstructor
 public class BankAccountService {
 
+    //TODO не видит бин
```

Comment:
To be honest, it is a general here. How to import Client, which will work ;)

Reply:
@ComponentScan works wonders